### PR TITLE
Feat: Add to playlist

### PR DESCRIPTION
**Pull request by munjalSaksham71**
feature: Add to playlist functionality and Screen

### CODE REVIEW

File: src/Screen/PlaylistScreen/PlaylistScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { Link } from "react-router-dom";
+
+const PlaylistScreen = () => {
+  const {
+    playlistState: { playlists },
+    playlistDispatch,
+  } = usePlaylist();
+  console.log(playlists);
+
+  const removeHandler = async (playlist_id, video_id) => {
```
Comment (mariyasada) [inline src/Screen/PlaylistScreen/PlaylistScreen.jsx]: remove console

**Review by mariyasada — APPROVED** [review]
The code structure is good. This Pr is good to merge

File: src/Screen/LoginScreen/LoginScreen.jsx [hunk 2]
```diff
@@ -13,12 +13,12 @@ const LoginScreen = () => {
     const submitHandler = async (e) => {
         e.preventDefault();
         if(email === "" || password === ""){
```
Comment (JantuDeb) [inline src/Screen/LoginScreen/LoginScreen.jsx]: `if(!email || !password){}`

File: src/Screen/VideoPreview/VideoPreview.jsx [hunk 3]
```diff
@@ -23,30 +23,35 @@ const VideoPreview = () => {
...
+        !isEmptyObject(video) &&
+          (await historyDispatch({ type: "ADD_TO_HISTORY", payload: video }));
       } catch (error) {
-        alert(error);
+        console.log(error);
       }
-    })()
-  }, [video])
+    })();
+  }, [video]);
```
Comment (JantuDeb) [inline src/Screen/VideoPreview/VideoPreview.jsx:50]: You may also check user is logged in  or not

**Review by JantuDeb — APPROVED** [review]
Overall it looks fine

File: src/Screen/VideoPreview/VideoPreview.jsx (hunk 3 above)
Comment (munjalSaksham71) [reply src/Screen/VideoPreview/VideoPreview.jsx:50]: For that- I already redirect user to home page even if the user wants to go login page(if loggedIn).

**Pull request merged**